uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1242,6 +1242,80 @@ sub computeSNR {
 }
 
 ################################################################
+##########  Order Imaging Modalities By Acq ####################
+################################################################
+
+sub orderModalitiesByAcq {
+
+    my $this = shift;
+    my ($row, $file, $acqProtID, $dataArr, $cmd, $message, $sth);
+    my ($tarchiveID, $tarchive_srcloc)= @_;
+    my $upload_id = getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc);
+
+    my $query = "SELECT DISTINCT f.AcquisitionProtocolID ".
+		"from files f ";
+    my $where = "WHERE f.TarchiveSource=?";   
+    $query = $query . $where;
+
+
+    if ($this->{debug}) {
+        print $query . "\n";
+    }
+
+    my $acqArr = ${$this->{'dbhr'}}->prepare($query);
+    $acqArr->execute($tarchiveID);
+    # For each of the files having this AcquisitionProtocolID
+    # load the file object to get the series_number  
+    while (my $row = $acqArr->fetchrow_hashref()) {
+	$acqProtID = $row->{'AcquisitionProtocolID'};
+	$query = "SELECT f.FileID, f.ModalityOrder ".
```

Comment:
no, you understood it right. I misnamed it here (it was the original name I chose but then I updated it in Loris but forgot to update here). Thanks for the catch!
done now